### DISCUSSION TITLE
Unknown error from an entity

### DISCUSSION THREAD

**Discussion by Noya2Coco** [doctrine/doctrine2#11346] [Support Questions]
I started coding in Symfony a few months ago, I came across a very inaccurate error and I can't find anyone talking about it on the internet. ChatGPT and other AIs can't find a solution to my problem.
My error is coming from one (or more) ManyToMany relationships of an entity. Here's the error:
AssertionError
assert($mapping instanceof ManyToManyOwningSideMapping)
The problem arises when I want to use my ''Atelier'' entity. I share this entity with you and the others related to it. Thank you in advance for your help.
[Entities.zip](https://github.com/doctrine/orm/files/14550844/Entities.zip)

**Comment by Mat40100** [comment]
Hey, this is probably because you have an attribute or annotation **JoinColumn** or **joinTable** on the **reverse side** of the ManyToMany relation
